cmd/eksctl/info.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/info"
+)
+
+func infoCmd(cmd *cmdutils.Cmd) {
+	var output string
+
+	cmd.SetDescription("info", "Output the version of eksctl, kubectl and OS info", "")
+	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
+		fs.StringVarP(&output, "output", "o", "", "specifies the output format (valid option: json)")
+	})
+	cmd.CobraCommand.Args = cobra.NoArgs
+	cmd.CobraCommand.RunE = func(_ *cobra.Command, args []string) error {
+		switch output {
+		case "":
+			fmt.Printf("eksctl version: %s\n", info.GetVersion().EksctlVersion)
+			fmt.Printf("kubectl version: %s\n", info.GetVersion().KubectlVersion)
+			fmt.Printf("OS: %s\n", info.GetVersion().OS)
```

Comment:
Do you think we could just call `GetVersion` just the once in this block? Would be good to avoid execing to kubectl 3 times in a row?